lib/peeps.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'database_connection'
+
+class Peep
+  
+  attr_reader :id, :peep, :date, :time, :username
+  def initialize(id:, peep:, date:, time:, username:)
```

Comment:
When you get to methods with a larger number of arguments like this it can be better to take in a single hash argument and read from that. Means argument order doesn't matter and allows easy add and subtract of arguments as things change without directly breaking existing connectivity.